app/src/components/ArticlesPublications.jsx

```diff
@@ -20,6 +20,10 @@ class ArticlesPublications extends Component {
         className={PubArticleStyle['c-publication-article']}
         key={articlePublication.link}
       >
+        <div
+          className={PubArticleStyle.image}
+          style={{ backgroundImage: 'url(' + articlePublication.image + ')' } }
```

Comment:
The linter complains here. Please figure out why your linter is not detecting these